src/triggers/index.js

```diff
@@ -259,3 +260,14 @@ export async function updatePost(post_uuid, post_fields) {
     getStore().dispatch(addError(e.message));
   }
 }
+
+export async function loadSchools() {
+  try {
+    let result = await client.schools();
+    getStore().dispatch(setSchools(result));
+
+    return result;
+  } catch (e) {
+    getStore().dispatch(addError(e.message));
+  }
+}
```

Comment:
final new-line is missing. please configure your editor to create one automatically